### PR TITLE
vlan member tagging / untagging issue #38

### PR DESCRIPTION
**Pull request by rahul-1475**
fixes orca_backend#38

- replaced openconfig url with sonic url to add vlan members.
- added new test case to check if_mode update.

### CODE REVIEW

File: orca_nw_lib/vlan.py [hunk 1]
```diff
@@ -89,6 +87,7 @@ def _create_vlan_db_obj(device_ip: str, vlan_name: str = None):
             if v.name == item.get("name"):
                 members.append(item)
         vlans_obj_vs_mem[v] = members
+        print(members)
```
Comment (kamalkrbh) [inline orca_nw_lib/vlan.py:90]: @rahul-1475 , Print can be removed. Use proper logging instead if needed.
```suggestion
        print(members)
```

File: orca_nw_lib/vlan_gnmi.py [hunk 2]
```diff
@@ -163,16 +162,16 @@ def del_vlan_from_device(device_ip: str, vlan_name: str):
...
-    autostate: VlanAutoState = None,
-    ip_addr_with_prefix: str = None,
-    anycast_addr: str = None,
-    enabled: bool = None,
-    descr: str = None,
-    mem_ifs: dict[str:IFMode] = None,
-    mtu: int = None,
+        device_ip: str,
+        vlan_name: str,
+        vlan_id: int,
```
Comment (kamalkrbh) [inline orca_nw_lib/vlan_gnmi.py:167]: @rahul-1475 , remove unused parameter.

File: orca_nw_lib/vlan_gnmi.py [hunk 3]
```diff
@@ -453,3 +452,68 @@ def remove_anycast_addr_from_vlan_on_device(
...
+    req = []
+    for if_name, if_mode in mem_ifs.items():
+        req.append(
+            {
+                "name": vlan_name,
+                "ifname": if_name,
+                "tagging_mode": "tagged" if if_mode == IFMode.TRUNK else "untagged",
+            }
+        )
+    path = get_sonic_vlan_base_path()
```
Comment (kamalkrbh) [inline orca_nw_lib/vlan_gnmi.py:497]: @rahul-1475 . Use get_gnmi_path from gnmi_util.py for creating paths.

Comment (rahul-1475) [reply orca_nw_lib/vlan_gnmi.py:497]: seen and resolved line to get_gnmi_path("sonic-vlan:sonic-vlan/VLAN_MEMBER/VLAN_MEMBER_LIST")

File: orca_nw_lib/vlan_gnmi.py (hunk 2 above)
Comment (rahul-1475) [reply orca_nw_lib/vlan_gnmi.py:167]: removed unused parmeter vlan_id

File: orca_nw_lib/vlan.py (hunk 1 above)
Comment (rahul-1475) [reply orca_nw_lib/vlan.py]: removed print

**Comment by rahul-1475** [comment]
other pr's

https://github.com/STORDIS/orca_backend/pull/41

**Pull request merged**